source/reference/json/cloud-backup-schedule-config-file.txt

```diff
@@ -0,0 +1,241 @@
+         snapshot occurs on the first day of the month and ``40``
+         indicates the last day of the month.
+
+       - In a weekly policy item, ``1`` indicates that the weekly
+         snapshot occurs on Monday and ``7`` indicates Sunday.
+
+       - In an hourly policy item, you can set the frequency interval to
+         ``1``, ``2``, ``4``, ``6``, ``8``, or ``12``. For hourly policy
+         items for NVMe clusters, MongoDB Cloud only accepts ``12`` as
+         the frequency interval value.
+
+   * - ``policies.``
+       ``policyItems.``
+       ``frequencyType``
+     - string
+     - Human-readable label that identifies the frequency type
+       associated with the backup policy.
+
+   * - ``policies.``
+       ``policyItems.``
+       ``retentionUnit``
+     - string
+     - Unit of time in which MongoDB Cloud measures snapshot retention.
+
+   * - ``policies.``
+       ``policyItems.``
+       ``retentionValue``
+     - integer
+     - Duration in days, weeks, or months that MongoDB Cloud retains the
+       snapshot. For less frequent policy items, MongoDB Cloud requires
+       that you specify a value greater than or equal to the value
+       specified for more frequent policy items. 
+
+       For example: If the hourly policy item specifies a retention of two
+       days, you must specify two days or greater for the retention of the
+       weekly policy item. 
+
+   * - ``referenceHourOfDay``
+     - integer
+     - Hour of day in Coordinated Universal Time (UTC) that represents
+       when MongoDB Cloud takes the snapshot.
+
+   * - ``referenceMinuteOfHour``
+     - integer
+     - Minute of the ``referenceHourOfDay`` that represents when MongoDB
+       Cloud takes the snapshot.
+
+   * - ``restoreWindowDays``
+     - integer
+     - Number of previous days that you can restore back to with
+       Continuous Cloud Backup accuracy. You must specify a positive,
+       non-zero integer. This parameter applies to continuous cloud
+       backups only.
+
+   * - ``updateSnapshots``
+     - boolean
+     - Flag that indicates whether to apply the retention changes in the
+       updated backup policy to snapshots that MongoDB Cloud took
+       previously.
+
+   * - ``useOrgAndGroupNamesInExportPrefix``
+     - boolean
+     - Flag that indicates whether to use organization and project names
+       instead of organization and project UUIDs in the path to the
+       metadata files that MongoDB Cloud uploads to your |aws| bucket. 
+
+For a full list of available settings, see the API documentation for
+:oas-atlas-op:`Update Cloud Backup Schedule for One Cluster </cloudProvider>`.
+
+.. _example-cloud-backup-schedule-config-file:
+
+Example Cloud Backup Schedule Configuration File
+------------------------------------------------
+
+To update a cloud backup schedule for a single cluster, specify the same
+service provider for your ``regionConfigs`` objects as shown in the
+following example file:
```

Comment:
Change: I'm confused by this statement.
1. I don't see a ``regionConfigs`` object in this example.
2. Why would I use the same service provider in the example (which is AWS) if I am using GCP or Azure?

Reply:
Sorry! I copied this over from the other page in this section and also forgot to change this 😅 Users should only be specifying fields they want updated.